src/server/util/spec_resolvers.cc

```diff
@@ -32,6 +32,10 @@ DEFINE_string(meta, "etcd", "Metadata storage, can be one of: etcd, local");
 DEFINE_string(etcd_endpoint, "http://127.0.0.1:2379", "endpoint of etcd");
 DEFINE_string(etcd_prefix, "vineyard", "path prefix in etcd");
 DEFINE_string(etcd_cmd, "", "path of etcd executable");
+DEFINE_string(spill_path, "/tmp/spill_path/",
+              "path of spilling temporary files");
```

Comment:
By default the spill shouldn't be enabled so please change the default value to `""`.